L1Trigger/VertexFinder/src/VertexFinder.cc

```diff
@@ -1122,11 +1123,11 @@ namespace l1tVertexFinder {
       // Chris' Quantised Network: Use values from L1GTTInputProducer pT, MVA1, eta
       auto& gttTrack = gttTracks_.at(counter);
       int pTBit = gttTrack.getTTTrackPtr()->getRinvBits() < 16383 ? gttTrack.getTTTrackPtr()->getRinvBits() : gttTrack.getTTTrackPtr()->getRinvBits()-16384;
-      float etaBit = gttTrack.getTTTrackPtr()->getTanlBits() < 32767 ? (gttTrack.getTTTrackPtr()->getTanlBits()+32767)/2 : (gttTrack.getTTTrackPtr()->getTanlBits()-32767)/2;
-      inputTrkWeight.tensor<float, 2>()(0, 0) = float(std::clamp(pTBit, 0, 4096))/4096.;
-      inputTrkWeight.tensor<float, 2>()(0, 1) = gttTrack.MVA1();
+      float etaBit = gttTrack.getTTTrackPtr()->getTanlBits() < 32767 ? (gttTrack.getTTTrackPtr()->getTanlBits()+32767)/2 : (gttTrack.getTTTrackPtr()->getTanlBits()-32767);
```

Comment:
As you now do /65534 below, we should get rid of the /2 if getTanlBits() < 32767

Reply:
Fixed